aws-iot-accountauditconfiguration/src/test/java/com/amazonaws/iot/accountauditconfiguration/CreateHandlerTest.java

```diff
@@ -104,7 +97,7 @@ public void handleRequest_AllPropertiesFilledOut_FirstTimeCreate_VerifyInteracti
     }
 
     @Test
-    public void handleRequest_DescribeShowsIdenticalConfig_ExpectSuccess_NoUpdateCall() {
+    public void handleRequest_DescribeShowsSameRoleArn_ExpectRAE() {
```

Comment:
Minor: let's add a comment like `the CreateHandler throws RAE as soon as it sees a RoleArn, no matter whether it's the same or not`, to avoid confusion.

Reply:
Done